prow/plugins/milestone/milestone.go

```diff
@@ -33,9 +34,9 @@ import (
 const pluginName = "milestone"
 
 var (
-	milestoneRegex   = regexp.MustCompile(`(?m)^/milestone\s+(.+)$`)
+	milestoneRegex   = regexp.MustCompile(`(?m)^/milestone\s+(.+?)\s*$`)
 	mustBeSigLead    = "You must be a member of the [%s](https://github.com/orgs/%s/teams/%s/members) github team to set the milestone."
-	invalidMilestone = "The provided milestone is not valid for this repository.  Here are the available milestones:\n %s"
+	invalidMilestone = "The provided milestone is not valid for this repository. Milestones in this repository: [%s]\n\nUse `/milestone %s` to clear the milestone."
 	clearKeyword     = "clear"
```

Comment:
unrelated to this PR, but shouldn't these strings be const?

Reply:
Yes they should be.
/shrug